tests/documentation/DocGrammarBearTest.py

```diff
@@ -0,0 +1,127 @@
+from queue import Queue
+import unittest
+import shutil
+
+from coalib.results.Diff import Diff
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import execute_bear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from bears.documentation.DocGrammarBear import DocGrammarBear
+from string import Template
```

Comment:
We have an unofficial imports style (absent in most bears) in which we do the bear imports at the bottom. You can have a glance over the PEP8 import style. :wink: